src/pages/Home/HomePage.tsx

```diff
@@ -10,8 +10,13 @@ import { Container, SelectContainer, WidgetWrapper } from './HomePage.styled';
 import { HomePageTestIds } from './HomePageTestIds';
 
 const HomePage = () => {
+  const [chosenCity, setChosenCity] = useState('');
+
   const [savedLocation, setSavedLocation] = useLocalStorage('current_location');
```

Comment:
**suggestion** unify names of thiose variables. savedLocation => currentLocation, setSavedLocation => setCurrentLocation

Reply:
Done